### PR TITLE
chore: Disable fetching new emails after mailbox error

### PR DESCRIPTION
**Pull request by sojan-official**
- disabled email fetch job if credentials for the channel isn't working
- notify customers when the email channel isn't working

fixes: https://github.com/chatwoot/chatwoot/issues/4174

### CODE REVIEW

File: app/mailers/administrator_notifications/channel_notifications_mailer.rb [hunk 1]
```diff
@@ -15,6 +15,14 @@ def facebook_disconnect(inbox)
     send_mail_with_liquid(to: admin_emails, subject: subject) and return
   end
 
+  def email_disconnect(inbox)
+    return unless smtp_config_set_or_development?
+
+    subject = 'Your Email Inbox configuration is not working'
```
Comment (pranavrajs) [inline app/mailers/administrator_notifications/channel_notifications_mailer.rb:21]: Your email inbox has been disconnected. Please update the credentials for SMTP/IMAP.

File: app/views/mailers/administrator_notifications/channel_notifications_mailer/email_disconnect.liquid [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<p>Hello,</p>
+
+<p>Your Email Inbox configuration is not working. </p>
```
Comment (pranavrajs) [inline app/views/mailers/administrator_notifications/channel_notifications_mailer/email_disconnect.liquid:3]: Your email inbox has been disconnected due to configuration errors. Please update it to continue receiving messages in <brand_name>/Chatwoot.

**Comment by sojan-official** [comment]
@aswindevps  if you have an IMAP email set up could you verify that this doesn't break anything ?

**Comment by aswindevps** [comment]
@sojan-official Tested below scenarios in local and staging.

1. Email conversation continuity.
2. Re-verification email on authorization error.

Screenshot 2022-03-22 at 11 30 07 AM


Working as expected.

**Review by aswindevps — APPROVED** [review]
LGTM

**Pull request merged**